docs/cli/gittuf_clone.md

```diff
@@ -9,8 +9,9 @@ gittuf clone [flags]
 ### Options
 
 ```
-  -b, --branch string   specify branch to check out
-  -h, --help            help for clone
+  -b, --branch string    specify branch to check out
+  -h, --help             help for clone
+      --root-key paths   path(s) of root keys to verify when cloning the repo. For gpg or fulcio key's pass in the key itself instead of the path to the key
```

Comment:
@lukpueh with your changes on ssh key support, we'll still be passing in a path to a key, right?

Reply:
I think this help message needs more work.
1) we need something slightly more generic than `paths` for the type to account for gpg/fulcio
2) the "short" help msg is just too long, I wonder if renaming the type could make life easier

Perhaps the type can be something like `public-key` and the message can just focus on the root keys bit. Is it okay for the help on `public-key` itself to indicate it can be a path or a gpg fingerprint or fulcio identity?

cc @flandweber @nealmcb I believe both of you had thoughts about the original parameter type and help message, so your input is appreciated!